### PR TITLE
send version to website

### PR DESCRIPTION
**Pull request by goldylucks**

### CODE REVIEW

File: chrome/manifest.dev.json [hunk 1]
```diff
@@ -11,6 +11,14 @@
   "background": {
     "page": "background.html"
   },
+  "externally_connectable": {
+    "matches": [
+      "http://localhost/*",
+      "http://home.parity/",
```
Comment (tomusdrw) [inline chrome/manifest.dev.json]: Should be "*.parity" (we already have local dapps support)

Comment (tomusdrw) [inline chrome/manifest.dev.json]: Also `http://localhost:*/*` ?

File: chrome/manifest.dev.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
   },
   "externally_connectable": {
     "matches": [
+      "http://127.0.0.1/*",
```
Comment (tomusdrw) [inline chrome/manifest.dev.json]: No need to specify any ports here?

File: chrome/manifest.dev.json [hunk 3]
```diff
@@ -13,6 +13,7 @@
...
       "http://localhost/*",
       "http://home.parity/",
```
Comment (tomusdrw) [inline chrome/manifest.dev.json]: This should be just `http://parity/` as far as I understand docs.

File: chrome/manifest.dev.json (hunk 2 above)
Comment (goldylucks) [inline chrome/manifest.dev.json]: nope, test both localhost and 127... like that and it works

File: chrome/manifest.dev.json (hunk 3 above)
Comment (goldylucks) [inline chrome/manifest.dev.json]: Tried:
`http://parity/`
`http:///parity/`
`http://*.parity/`

no luck

Comment (tomusdrw) [inline chrome/manifest.dev.json]: https://developer.chrome.com/extensions/manifest/externally_connectable#manifest See example.

Comment (goldylucks) [inline chrome/manifest.dev.json]: these?
```
"https://*.google.com/*",
"*://*.chromium.org/*",
```

couldn't get it to work with `.parity` ending

File: chrome/manifest.dev.json [hunk 4]
```diff
@@ -11,6 +11,15 @@
   "background": {
     "page": "background.html"
   },
+  "externally_connectable": {
+    "matches": [
+      "http://127.0.0.1/*",
+      "http://localhost/*",
+      "http://home.parity/",
+      "http://parity.parity/",
```
Comment (tomusdrw) [inline chrome/manifest.dev.json]: Should include `status.parity` too.

**Comment by tomusdrw** [comment]
looks good, merge at will.

**Pull request merged**